CefSharp/IValue.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright © 2010-2018 The CefSharp Authors. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+using System;
+using System.Collections.Generic;
+using CefSharp.Enums;
+using ValueType = CefSharp.Enums.ValueType;
+
+namespace CefSharp
+{
+    /// <summary>
+    /// Interface representing CefValue.
+    /// </summary>
+    public interface IValue : IDisposable
+    {
+        ValueType Type { get; }
+
+        /// <summary>
+        /// Returns the underlying value as type bool.
+        /// </summary>
+        /// <returns></returns>
+        bool GetBool();
```

Comment:
Add missing return comment,  same for the other methods.

Reply:
Looks like in all these methods `returns` comments will be the same as `summary` comments, I don't know what should I add.

Reply:
Duplicate is fine, they'll appear in the published `API Doc` (http://cefsharp.github.io/api/)